package.json

```diff
@@ -30,10 +30,13 @@
     "@types/react": ">=16.0 <18.0",
     "@types/react-dom": "^17.0.9",
     "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.18.0",
```

Comment:
All this was needed to make the tests run. Looks like nobody's run them in awhile.